Source/WebCore/dom/DocumentMarker.h

```diff
@@ -124,6 +125,9 @@ class DocumentMarker : public CanMakeWeakPtr<DocumentMarker> {
         State state { State::Pending };
     };
 #endif
+    struct TransparentContentData {
+        float alphaValue { 0.0 };
+    };
```

Comment:
Since this isn't supported yet, I'd just leave it out of this PR altogether. We can reintroduce this once this can become unified with `DraggedContentRanges`.

(Then you can also remove the `FIXME` below)